sql/src/main/java/org/apache/druid/sql/calcite/schema/DruidSchema.java

```diff
@@ -375,11 +373,7 @@ private void startCacheExec()
   @LifecycleStart
   public void start() throws InterruptedException
   {
-    if (druidSchemaManager == null || druidSchemaManager instanceof NoopDruidSchemaManager) {
-      startCacheExec();
-    } else {
-      initialized.countDown();
-    }
+    startCacheExec();
```

Comment:
What was the reasoning behind disabling the cache when schemaManager is enabled in first place?